src/css/custom.css

```diff
@@ -470,7 +470,8 @@ a .ngrok--card svg {
  * operation: make the banner smaller
 */
 #lanyard_root [class^="content"] {
-  padding-block: 12px;
+  padding-top: 12px;
+  padding-bottom: 12px;
```

Comment:
padding-block has less browser support than I would like for a global/public facing site 😞